### ISSUE TITLE
Cannot install marathon 1.3.0 package on jessie (debian 8.4)

### ISSUE THREAD

**Issue by rhruiz** [mesosphere/marathon#4409]
The package is not listed when using `apt-get update`. The last version available is 1.1.2.

```
root@*******:~# cat /etc/apt/sources.list.d/mesosphere.list 
deb http://repos.mesosphere.io/debian jessie main
```

And then:

```
root@*******:~# apt-get update -qq && apt-cache show marathon | grep 1\\.3
root@*******:~# 
```

When downloading individual `deb`s from https://repo.mesosphere.com, I get errors as:

```
root@*******:~#  wget http://repos.mesosphere.com/debian/pool/main/m/marathon/marathon_1.3.0-1.0.505.debian81_amd64.deb
root@*******:~# dpkg -i marathon_1.3.0-1.0.505.debian81_amd64.deb 
(Reading database ... 58828 files and directories currently installed.)
Preparing to unpack marathon_1.3.0-1.0.505.debian81_amd64.deb ...
Unpacking marathon (1.3.0-1.0.505.debian81) over (1.3.0-1.0.496.debian81) ...
dpkg-deb (subprocess): cannot copy archive member from 'marathon_1.3.0-1.0.505.debian81_amd64.deb' to decompressor pipe: unexpected end of file or stream
dpkg-deb: error: subprocess paste returned error exit status 2
dpkg: error processing archive marathon_1.3.0-1.0.505.debian81_amd64.deb (--install):
 cannot copy extracted data for './usr/bin/marathon' to '/usr/bin/marathon.dpkg-new': unexpected end of file or stream
Errors were encountered while processing:
 marathon_1.3.0-1.0.505.debian81_amd64.deb
```

**Comment by jasongilanfarr** [comment]
We're investigating the packaging. There is definitely something wrong with how we are publishing deb files.

**Comment by drcrallen** [comment]
is `1.0.505` the package for the marathon 1.3.0 release?

**Comment by jasongilanfarr** [comment]
Unfortunately, I don't know and I'm the only one awake (more or less) at the moment.

**Comment by Daniel-ltw** [comment]
related? 

https://github.com/mesosphere/marathon/issues/4391

**Comment by eromoe** [comment]
Can not install on centos 6.5 too:

```
Total download size: 8.0 M
Installed size: 89 M
Downloading Packages:
marathon-1.3.0-1.0.506.el6.x86_64.rpm                                                                             | 8.0 MB     01:38
Running rpm_check_debug
Running Transaction Test
Transaction Test Succeeded
Running Transaction
  Installing : marathon-1.3.0-1.0.506.el6.x86_64                                                                                     1/1
Error unpacking rpm package marathon-1.3.0-1.0.506.el6.x86_64
error: unpacking of archive failed on file /usr/bin/marathon;57df41dd: cpio: read
  Verifying  : marathon-1.3.0-1.0.506.el6.x86_64                                                                                     1/1

Failed:
  marathon.x86_64 0:1.3.0-1.0.506.el6

Complete!
```

**Comment by liquid-sky** [comment]
Same is true for Ubuntu (Trusty), but the latest package available there is `1.3.0-1.0.505.ubuntu1404`, which is RC7 afaik. 

```
apt-get install marathon=1.3.0-1.0.505.ubuntu1404
Reading package lists... Done
Building dependency tree
Reading state information... Done
The following NEW packages will be installed:
  marathon
0 upgraded, 1 newly installed, 0 to remove and 11 not upgraded.
Need to get 2441 kB of archives.
After this operation, 92.9 MB of additional disk space will be used.
Get:1 http://repos.mesosphere.io/ubuntu/ trusty-testing/main marathon amd64 1.3.0-1.0.505.ubuntu1404 [2441 kB]
Fetched 2441 kB in 3s (738 kB/s)
E: Archive is too short
E: Prior errors apply to /var/cache/apt/archives/marathon_1.3.0-1.0.505.ubuntu1404_amd64.deb
debconf: apt-extracttemplates failed: No such file or directory
Selecting previously unselected package marathon.
(Reading database ... 16006 files and directories currently installed.)
Preparing to unpack .../marathon_1.3.0-1.0.505.ubuntu1404_amd64.deb ...
Unpacking marathon (1.3.0-1.0.505.ubuntu1404) ...
dpkg-deb (subprocess): cannot copy archive member from '/var/cache/apt/archives/marathon_1.3.0-1.0.505.ubuntu1404_amd64.deb' to decompressor pipe: unexpected end of file or stream
dpkg-deb: error: subprocess paste returned error exit status 2
dpkg: error processing archive /var/cache/apt/archives/marathon_1.3.0-1.0.505.ubuntu1404_amd64.deb (--unpack):
 cannot copy extracted data for './usr/bin/marathon' to '/usr/bin/marathon.dpkg-new': unexpected end of file or stream
Processing triggers for ureadahead (0.100.0-16) ...
Errors were encountered while processing:
 /var/cache/apt/archives/marathon_1.3.0-1.0.505.ubuntu1404_amd64.deb
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

**Comment by lloesche** [comment]
I'm posting status updates to this issue in https://github.com/mesosphere/marathon/issues/4391#event-793773869
